platform/android/export/export.cpp

```diff
@@ -2571,13 +2640,16 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 
 			for (int i = 0; i < icon_densities_count; ++i) {
```

Comment:
Add the `use_custom_build` check to skip the steps below if custom build is enabled.